tools/phenomenal/isatools/slicer/isa_get_data_files_collection.xml

```diff
@@ -0,0 +1,186 @@
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" level="fatal"/>
+    </stdio>
+
+    <command><![CDATA[
+        run_mtblisa.py isa-tab-get-data-collection
+        "${isatab_input.extra_files_path}" "./"
+        --json-query "{ \"$factor_name\": \"$factor_value\" }"
+]]>
+    </command>
+    <inputs>
+	    <param format="isa-tab" name="isatab_input" type="data" label="ISA-Tab"
+               help="An ISA-Tab datatype history item"/>
+        <param name="factor_name" type="text" label="Factor name to filter on."
+               help="Factor name relevant to the specified MetaboLights study,
+               e.g. genotype. You can get a study's factor names by using the
+               study_factors tool."/>
+        <param name="factor_value" type="text"
+               label="Factor value to filter on."
+               help="Factor value relevant to the specified MetaboLights study,
+               e.g. cyp79. You can get a study's factor values by using the
+               study_factor_values tool."/>
+    </inputs>
+    <outputs>
+        <collection type="list" label="Factor-sliced data file collection" name="raw_files">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.[a-zA-Z0-9]+$"/>
+        </collection>
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="mtbls_id" value="MTBLS1"/>
+            <param name="factor_name" value="Gender"/>
+            <param name="factor_value" value="Female"/>
+            <output name="output"/>
+        </test>
+    </tests>
+
+    <help>
+<!-- @@@BEGIN_RST@@@ -->
+========================================
+Get a MetaboLights study data files list
+========================================
+
+This tool returns a list of data files sliced on a filtered factor name
+and value.
+
+-----
+Input
+-----
+
+ISA-Tab input source
+====================
+Choose an input source, either direct from Metabolights or from an ISA-Tab
+history item that has already been downloaded or created in Galaxy.
+
+MetaboLights Study Identifier
+=============================
+If the input source is Metabolights, then provide the MTBLS identifier
+available at MetaboLights, e.g. MTBLS2.
+
+ISA-Tab datatype history item
+=============================
+If the input source is an ISA-Tab history item, select from existing data
+history items. ISA-Tab items can be created by using the Metabolights
+downloader or the ISA-Tab metadata creation tools.
+
+Factor name to filter on
+========================
+Factor name relevant to the specified MetaboLights study, e.g.
+replicate. You can get the relevant factor names for a particular study
+by using the `study_factors` tool.
+
+Factor value to filter on
+=========================
+Factor value relevant to the specified MetaboLights study, e.g. Exp2.
+You can get the relevant factor values for a particular study and factor
+by using the `study_factor_values` tool.
+
+------
+Output
+------
+
+List of data files as JSON
```

Comment:
This is incorrect for this tool, right? it gives you a dataset collection, not json with paths.

Reply:
I leave it to you, squash and merge if you are happy to go ahead (please keep suggested # with PR number that github will put in the commit name).

Thanks @djcomlab!

Reply:
Oh yes, that's right. Let me double check the help sections now...